hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -299,7 +299,7 @@ public String getBootstrapIndexClass() {
 
   public static String getDefaultBootstrapIndexClass(Properties props) {
     String defaultClass = BOOTSTRAP_INDEX_CLASS_NAME.defaultValue();
-    if ("false".equalsIgnoreCase(props.getProperty(BOOTSTRAP_INDEX_ENABLE.key()))) {
+    if (!(Boolean)props.getOrDefault(BOOTSTRAP_INDEX_ENABLE.key(), BOOTSTRAP_INDEX_ENABLE.defaultValue())) {
```

Comment:
IMO, you could use `HoodieConfig#getBooleanOrDefault` instead of this.

Reply:
+1, s.g.

Reply:
+1